### ISSUE TITLE
sessions does not work well with heroku multiple dynos

### ISSUE THREAD

**Issue by yrxwin** [balderdashy/sails#2205]
I use sails 0.10.1, and I set all the config like the sails documents said (use redis for session).

But when I use two dynos, the sessions do not work across dyno.

**Comment by CWyrtzen** [comment]
is there any follow up with this?

**Comment by sailsbot** [comment]
Thanks for posting, @yrxwin.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
